README.md

```diff
@@ -1,56 +1,92 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# HTML/CSS & JavaScript capstone project - Conference page
 
+> The projects builds an online website for a comedy show. The design is an original by from the [Cindy Shin in Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) of a conference website. Original design idea by Cindy Shin in Behance.
 
 ## Built With
 
-- Major languages
-- Frameworks
+- Major languages : css, html, js
 - Technologies used
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+## Live Demo
 
+[My Website](https://manq2010.github.io/HTML-CSS-JavaScript-capstone-project-1/)
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+To get a local copy up and running follow these simple example steps.
 
+## Getting Started
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+- Node.js installed
+
 ### Setup
 
-### Install
+#### Set-up GitHub Actions
```

Comment:
[OPTIONAL] 
- [ ] Kindly you can consider walking the user through the process of getting the local copy of the project  form GitHub
`hint: You may say: git clone <repos link> `
`cd repos <repos directory_name>`
`open in IDE`
`Go live with live server`
- [ ] You may also consider doing the same for the installation, setup and prerequisites, `Be as much detailed as possible.`